src/Entity/LineItem.php

```diff
@@ -134,6 +137,18 @@ public function isAvailableForDate(DateTimeInterface $date): bool
         return $this->startAt <= $date && $this->endAt >= $date;
     }
 
+    public function getMaxAttempts(): ?int
```

Comment:
And in case if previous proposals would be accepted (and as soon as it doesn't makes much sense to have null for infinite attempts), please, change the return type hint of this method